src/cli.ts

```diff
@@ -1,33 +1,78 @@
 #!/usr/bin/env node
 
+import fs from 'fs/promises';
+import path from 'path';
+import crypto from 'crypto';
+import dotenv from 'dotenv';
 import { Command } from "commander";
-import { authenticate } from "./commands/auth";
-import { install } from "./commands/install";
-import { deploy } from "./commands/deploy";
-import figlet from "figlet";
+import { authenticate, checkAuth } from "./commands/auth.js";
+import { install } from "./commands/install.js";
+import { deploy } from "./commands/deploy.js";
+import figlet from 'figlet';
 
-console.log(figlet.textSync("Anyflow CLI"));
+async function ensureEnvFile() {
+  const envPath = path.resolve(process.cwd(), '.env');
+  try {
+    await fs.access(envPath);
+  } catch {
+    const key = crypto.randomBytes(32).toString('hex');
+    await fs.writeFile(envPath, `ENCRYPTION_KEY=${key}`);
+    console.log('Created new .env file with encryption key.');
+  }
+}
 
-const program = new Command();
+async function init() {
```

Comment:
Is `init()` command necessary? I think it's not.

By default, `main()` function is called and the first thing it does is call the same functions as `init()` does.